spec/support/bolt_configuration.rb

```diff
@@ -7,8 +7,8 @@
     solidus_bolt_configuration.bearer_token = ENV['BOLT_BEARER_TOKEN']
     solidus_bolt_configuration.environment = 'sandbox'
     solidus_bolt_configuration.merchant_public_id = ENV['BOLT_MERCHANT_PUBLIC_ID']
-    solidus_bolt_configuration.merchant_id = ENV['BOLT_MERCHANT_ID']
-    solidus_bolt_configuration.api_key = ENV['BOLT_API_KEY']
+    solidus_bolt_configuration.division_public_id = ENV['BOLT_DIVISION_PUBLIC_ID'] || 'UwZWGwaPAuFB'
+    solidus_bolt_configuration.api_key = ENV['BOLT_API_KEY'] || 'db1bd1ff3631f9c3e30d3555ad2989fa7cbed58b2344aea911aa7e2d52dba3e3'
```

Comment:
Why you have provided fallback credentials here?

Reply:
silly mistake
fixing it now

Reply:
fixed now